sources/packages/backend/apps/api/src/route-controllers/institution-user/_tests_/e2e/institution-user.institutions.controller.createInstitutionUserWithAuth.e2e-spec.ts

```diff
@@ -0,0 +1,267 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import * as faker from "faker";
+import {
+  authorizeUserTokenForLocation,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAuthRelatedEntities,
+  getInstitutionToken,
+  InstitutionTokenTypes,
+  mockBCeIDAccountDetails,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeInstitutionLocation,
+  createFakeUser,
+} from "@sims/test-utils";
+import {
+  IdentityProviders,
+  Institution,
+  InstitutionLocation,
+  InstitutionUserTypes,
+} from "@sims/sims-db";
+import { TestingModule } from "@nestjs/testing";
+import {
+  BCEID_ACCOUNT_NOT_FOUND,
+  INSTITUTION_USER_ALREADY_EXISTS,
+} from "../../../../constants";
+
+describe("InstitutionUserInstitutionsController(e2e)-createInstitutionUserWithAuth", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let appModule: TestingModule;
+  let collegeFLocationA: InstitutionLocation;
+  let collegeFLocationB: InstitutionLocation;
+  let collegeFInstitution: Institution;
+
+  beforeAll(async () => {
+    const { nestApplication, module, dataSource } =
+      await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    appModule = module;
+    // College F.
+    const { institution: collegeF } = await getAuthRelatedEntities(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    // Location A.
+    collegeFLocationA = createFakeInstitutionLocation({
+      institution: collegeF,
+    });
+    // Location B.
+    collegeFLocationB = createFakeInstitutionLocation({
+      institution: collegeF,
+    });
+    await authorizeUserTokenForLocation(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+      collegeFLocationA,
+    );
+    await authorizeUserTokenForLocation(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+      collegeFLocationB,
+    );
+    collegeFInstitution = collegeFLocationA.institution;
```

Comment:
Minor, not a blocker, it can be associated from the previous variable. It seems easier to read, IMO.
```ts
collegeFInstitution = collegeF;
```